LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -12,14 +12,15 @@ public class SpinnerController : Controller
         //DIJ in 4 STEPS -
         //TODO: 0) Register the Repository class as a service in Program.cs 
         //TODO: 1) add an instance variable here of type Repository
-
+        Repository spins = new Repository();
```

Comment:
You only need to declare the instance variable here - you will assign it in the constructor